recipe/meta.yaml

```diff
@@ -1,46 +1,58 @@
+{% set python_min = '3.10' %}
 {% set name = "openfe" %}
-{% set version = "0.14.0dev0" %}
+{% set version = "1.5.0.dev0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   git_url: https://github.com/OpenFreeEnergy/{{ name }}.git
-  git_rev: e2eeb85356a5426a3d54aa2e61f6b01212ac770e
+  git_rev: 2d2fef2a07e4f0da50f881dddbad6c6af0d7c4d1
   git_depth: -1
 
 build:
   noarch: python
   number: 0
   entry_points:
     - openfe = openfecli.cli:main
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  script: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . -vv --no-deps
 
 requirements:
   host:
     - pip
-    - python >=3.9
-    - versioningit
+    - python {{ python_min }}
+    - setuptools
+    - setuptools_scm
   run:
+    - cinnabar ==0.5.0rc0
     - click
-    - gufe
-    - lomap2 >=2.1.0
+    - gufe ==1.5.0rc0
+    - kartograf ~=1.0
+    - lomap2 >=3.2.1
+    - matplotlib-base
     - networkx
-    - numpy 
-    - openff-toolkit
-    - openmm
-    - openmmforcefields >=0.11.2
-    - openmmtools >=0.23.1
+    - numpy
+    - openfe-analysis ~=0.3.1
+    - openff-toolkit >=0.16,<0.17
+    - openff-units==0.3.0=pyhd8ed1ab_1 # For some reason, it wants the _0 build with is broken
+    - openmmforcefields ==0.14.2
+    - openmm !=8.1.0,>=8.0.0
```

Comment:
do you want to do a `>8.2.0` pin here so we know it's pulling in at least 8.2 for this dev release?

Reply:
not a bad idea, I checked the output to make sure it was but since this is a dev build we can really do whatever we want

Reply:
If we're going to do back to back tests with different OpenMM versions for this RC, it may be best to not pin?

Reply:
Alternatively we can just make another build when it's needed

Reply:
I'm fine with releasing the pin if we're calling it "supports 8.2", but we should pin it if we're saying "this is the release with openmm 8.2"

Reply:
The intent of this RC is to the Collab notebooks to work again. I'd say "whatever minimal changes are necessary to allow for that" are the aim. The 8.2 support is a side effect of numpy 2.0 support rather than an aim imho.